site/app/views/grading/ElectronicGraderView.php

```diff
@@ -1197,6 +1197,7 @@ public function renderAutogradingPanel($version_instance, bool $show_hidden_case
         $this->core->getOutput()->addInternalJs('submission-page.js');
         $this->core->getOutput()->addInternalJs('drag-and-drop.js');
         $this->core->getOutput()->addVendorJs('bootstrap/js/bootstrap.bundle.min.js');
+        $this->core->getOutput()->addModuleJs("grading.js");
```

Comment:
Should this be `addInternalModuleJs` instead?

The `addModuleJS` function won't timestamp your given URL and also won't search the `mjs` directory for the file and causing a JS error.
Also should adding this file be in both the autograding and submission panel I think it would be sending this file over twice